### PR TITLE
IN-96 Add facilities-cdw app

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vasdvp.atlassian.net/browse/IN-96

### CODE REVIEW

**Comment by jblefkove-va** [comment]
This has successfully deployed to QA

File: products/facilities-cdw.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DU_ARTIFACT=health-apis-facilities-cdw-deployment
+DU_VERSION=0.0.14
```
Comment (jhulbert-va) [inline products/facilities-cdw.conf]: Should the `DU_VERSION` be 1.x.x?

Comment (jblefkove-va) [reply products/facilities-cdw.conf]: I was keeping it at 0.0.x while I experimented. Will go to 1.0.0 shortly.

**Comment by jhulbert-va** [comment]
Can you add tests to the `ingress.tests` file for the endpoints this application serves?

**Comment by jblefkove-va** [comment]
:shipit:

**Pull request merged**